apps/api/src/hyp3_api/api-spec/openapi-spec.yml

```diff
@@ -243,8 +243,13 @@ components:
       type: object
       additionalProperties: false
       properties:
+        start:
+          $ref: "#/components/schemas/datetime"
         end:
           $ref: "#/components/schemas/datetime"
+        intersectsWith:
+          type: string
+          format: wkt
```

Comment:
leaning slightly towards creating a stand-alone component for `intersectsWith` to avoid a tiny bit of duplication between updating/creating a subscription